src/components/collection/SimilarCollections.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useTranslation } from 'react-i18next';
+import Typography from '@material-ui/core/Typography';
+import Grid from '@material-ui/core/Grid';
+import { makeStyles } from '@material-ui/core/styles';
+import SimilarCollection from './SimilarCollection';
+import SimilarCollectionsHeader from './SimilarCollectionsHeader';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    flexGrow: 1,
+  },
+  paper: {
+    padding: theme.spacing(2),
+    textAlign: 'center',
+    color: theme.palette.text.secondary,
+  },
+}));
+
+function SimilarCollections({ similarCollections }) {
+  const { t } = useTranslation();
+  const classes = useStyles();
+
+  return (
+    <div className={classes.root}>
+      <SimilarCollectionsHeader />
+      {!similarCollections.length ? (
+        <div className="Main">
+          <Typography variant="h5" color="inherit">
+            {t('This collection is empty.')}
+          </Typography>
+        </div>
+      ) : (
+        <Grid container spacing={2}>
+          {similarCollections.map((similarCollection) => (
+            <Grid
+              key={similarCollection.id}
+              item
+              xs={12}
+              sm={4}
+              md={3}
+              lg={3}
+              xl={2}
+            >
+              <SimilarCollection similarCollection={similarCollection} />
+            </Grid>
+          ))}
+        </Grid>
+      )}
+    </div>
+  );
+}
+
```

Comment:
This should be adapted to the collections, not left commented out.

Reply:
Doesn't `yarn lint` complain that the prop types are not set correctly?

Reply:
You're right, `lint` does complain. I commented out this section initially and forgot to uncomment it. Let's discuss the purpose of .propTypes and how it works.